sale_procurement_analytic/models/product.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    def _need_procurement(self):
+        # When sale/product is installed alone, there is no need to create
```

Comment:
Yeah, I understand you, but this will create procurement for everything, which is not correct. This will only create procurements on MTO/dropshipping products. Make one thing, add the condition that this should be used only when testing the module (something similar to https://github.com/OCA/bank-payment/blob/8.0/account_payment_transfer_reconcile_batch/models/payment_order.py#L28 - with one condition less as you don't need to call the same method 2 times).